### DISCUSSION TITLE
“thiz"在接口中的实际意义是什么

### DISCUSSION THREAD

**Discussion by yqs112358** [Tencent/ScriptX#46] [Q&A] [answered]
不知道我的理解是否正确：
thiz表示当前CPP对象绑定的脚本对象
构造函数中给ScriptClass传入thiz是为了将新构造的对象初始化为现有的脚本对象的子对象

那在父对象销毁的时候，他所有的子对象也都会销毁吗

**Comment by LanderlYoung** [comment, marked as answer]
是的，ScriptClass构造的实际正式脚本对象构造的时候，因此`thiz`表示的表示的就是对应的脚本对象。
